### DISCUSSION TITLE
partial sort

### DISCUSSION THREAD

**Discussion by jacob-roth** [Morwenn/cpp-sort#222] [Q&A]
Hi! Is there any (current or planned) functionality for partial sorting? Say I only need have the smallest `k` of `n` (`k` << `n`) items sorted in increasing order.

**Comment by Morwenn** [comment]
Hi, there is currently no such feature as partial sort in the library, and no plan in the near future to work on that. I currently have my hands full working on the 2.0.0 release, with conceptification of the library among many other things, and the next goal will likely be to improve its tooling and implement more security features.

The truth is I'm working really slowly when I find the time, so unless I someday feel really strongly about adding partial sort, chances are that it won't happen in the next decade.
